client/my-sites/patterns/home/style.scss

```diff
@@ -1,9 +1,14 @@
 @import "@automattic/components/src/styles/typography";
-@import "@wordpress/base-styles/breakpoints.scss";
+@import "@wordpress/base-styles/breakpoints";
 
 .is-section-patterns {
 	.layout__content {
 		padding: 0;
+
+		.masterbar {
```

Comment:
Side note: it's somewhat annoying that the logged-out navigation is also called "master bar" 😕